reverse-proxy.md

```diff
@@ -271,6 +271,7 @@ server {
         proxy_set_header Host $host;
         proxy_set_header X-Real-IP $remote_addr;
         proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_read_timeout 86400s;
         client_max_body_size 0;
```

Comment:
btw, do we need both also for the other nginx based reverse proxies and what about the http2_body_preread_size?

Reply:
client_max_body_size:
- Original NPM has set it to [2GB](https://github.com/NginxProxyManager/nginx-proxy-manager/blob/fd30cfe98bba365e9630a791bdc86f01c08c6fa1/docker/rootfs/etc/nginx/nginx.conf#L36), it needs to be increased inside the advanced conf tab
- My NPM fork: https://github.com/ZoeyVid/nginx-proxy-manager/blob/21e765caa29cbf8987c5caf37d52096495488294/rootfs/usr/local/nginx/conf/nginx.conf#L29
- Synology no idea (my disk station seem to not support this proxy function)

proxy_read_timeout:
- Original NPM, needs to be set inside advanced tab
- My npm fork: https://github.com/ZoeyVid/nginx-proxy-manager/blob/21e765caa29cbf8987c5caf37d52096495488294/rootfs/usr/local/nginx/conf/nginx.conf#L46
- Synology no idea (my disk station seem to not support this proxy function)

And to http2_body_preread_size:
After reading [this](https://blog.cloudflare.com/delivering-http-2-upload-speed-improvements), I think we should instead set `client_body_buffer_size` to 256kb / 512kb

Reply:
Thanks Zoey! Lets handle this in a follow-up.